doc(Pages): Update remaining pages to remove reference to Portal (#1737)
bonitasoft/bonita-doc: modules/ROOT/pages/logging.adoc

> The components of Bonita have their own ways to produce logs, which are described below.

Reviewer: The components of Bonita => Bonita components

Reviewer: which are described below. => as described below.

> The components of Bonita have their own ways to produce logs, which are described below.

Reviewer 2: ```suggestion
Bonita components have their own ways to produce logs, as described below.
```

=== Bonita Runtime

> The Bonita Runtime is made out of two pars :

Reviewer: made out = make out = to deal with a situation, usually in a successful way. 

The Bonita Runtime is made out of two pars : = > The Bonita Runtime has two components:

> The Bonita Runtime is made out of two pars :

Reviewer 2: ```suggestion
The Bonita Runtime has two components :
```

> In Bonita Studio you have access to two log files:

Reviewer: to two log files: => to the following log files:

> In Bonita Studio you have access to two log files:

Reviewer 2: ```suggestion
In Bonita Studio you have access to the following log files :
```
